### PR TITLE
Rw actions testing

### LINKED ISSUE #36: Add actions testing
-One file needed for all actions

### LINKED ISSUE #37: Add reducers testing
-Separate files needed for each reducer

### PR DESCRIPTION
**Pull request by rwilliams659**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
- This adds all test coverage for our current 4 actions and current 4 reducers. 

- I also moved the collectDrinkData function out of actions since it was not an action creator & couldn't be tested as such. I moved it into the app component just below mapDispatchToProps, which seems to work and makes more sense. We can talk about if we need to move it somewhere else though. 

### Files modified:
- actions.test.js
- alcoholicDrinks-reducer.test.js
- drinksList-reducer.test.js
- error-reducer.test.js
- nonAlcoholicDrinks-reducer.test.js 
- App.js

### Has This Been Tested?
- [ ] No
- [X] Yes

### Are there specific methods for testing?
Run the test suite! The actions test file & the 4 reducers test file should be passing. 

### What are the relevant tickets?
resolve #36 
resolve #37 

### Message/Questions for reviewer:
As we move forward & add more actions & reducers, we should write the additional tests needed concurrently. 

### Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/actions/actions.test.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          strDrinkThumb: "https://g-and-t-img.com",
+          idDrink: '2'
+        }
+      ]
+    }
+
+    const result = actions.updateDrinksList(drinksList);
+
+    expect(result).toEqual(expectedAction);
+  });
```
Comment (relyt4me) [inline src/actions/actions.test.js:36]: Clean and simple

File: src/actions/index.js [hunk 2]
```diff
@@ -21,22 +19,6 @@ export const createNonAlcoholicDrinks = (nonAlcoholicDrinks) => {
...
-        } else {
-          dispatch(createNonAlcoholicDrinks(drinks.drinks))
-        }
-      })
-      .catch(error => {
-        dispatch(createError('We\'re sorry, our bar is closed!'))
-      })
-  }
-}
-
```
Comment (relyt4me) [inline src/actions/index.js:39]: I might need to go over why we are getting rid of this

File: src/reducers/error-reducer.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  });
+
+  it('should return the correct state if action is ERRORED', () => {
+    const initialState = 'No drinks found';
+    const action = {
+      type: 'ERRORED',
+      error: 'No drinks with that ingredient. Please try a different search!'
+    }
+
+    const newState = 'No drinks with that ingredient. Please try a different search!';
```
Comment (relyt4me) [inline src/reducers/error-reducer.test.js:18]: I like this naming convention for readability

**Review by relyt4me — APPROVED** [review]

File: src/actions/index.js (hunk 2 above)
Comment (tarynmartin) [reply src/actions/index.js:39]: Agreed. I feel like moving it into App defeats the purpose of seperating it like we did?

**Review by tarynmartin — APPROVED** [review]
Looks good, though I'm not sure about moving collecDrinksData back into App

**Pull request merged**